src/components/homepage/ProfileHighlight.astro

```diff
@@ -13,71 +14,17 @@ const featuredProfiles = await getFeaturedProfiles(featuredProfilePaths);
       <h2 class="text-3xl md:text-4xl font-bold">Meet Our Alumni</h2>
     </div>
 
-    {
-      Array.isArray(featuredProfiles) && featuredProfiles.length > 0 ? (
-        <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-8">
-          {featuredProfiles.slice(0, 3).map((alumn) => {
-            const { name, program, batch, image, slogan, tags, history } =
-              alumn.data;
-            return (
-              <div class="bg-gray-800 rounded-lg shadow-lg p-6 hover:shadow-xl transition-shadow duration-300">
-                {image?.url ? (
-                  <img
-                    src={image.url}
-                    alt={`Profile picture of ${name}`}
-                    class="w-full h-48 object-cover rounded-t-lg"
-                  />
-                ) : (
-                  <div class="w-full h-48 bg-gray-700 rounded-t-lg flex items-center justify-center">
-                    <span class="text-gray-400">No Image Available</span>
-                  </div>
-                )}
-                <div class="pt-4 text-center">
-                  <h3 class="text-xl font-semibold">{name}</h3>
-                  <span class="text-sm">
-                    {program}, Class of {batch}
-                  </span>
-                  {slogan && <p class="text-gray-400 italic">{slogan}</p>}
-                  <div class="flex space-x-2 mt-4 ">
-                    {tags.map((tag, index) => (
-                      <span
-                        style={{ marginRight: "0.5rem", marginLeft: "0.5rem" }}
-                        class="bg-blue-200 text-blue-800 text-xs font-semibold px-2 py-1 rounded">
-                        {tag}
-                      </span>
-                    ))}
-                  </div>
-                  <div class="mt-4 text-left">
-                    <ul class="text-sm">
-                      {history.map((item, index) => (
-                        <li>
-                          <strong>{item.year}:</strong> {item.position} at{" "}
-                          {item.company}
-                        </li>
-                      ))}
-                    </ul>
-                  </div>
-                  <div class="mt-6 text-center">
-                    <a
-                      href={`/profiles/${alumn.slug}`}
-                      class="text-blue-500 hover:underline"
-                    >
-                      See More
-                    </a>
-                  </div>
-                </div>
-              </div>
-            );
-          })}
-        </div>
-      ) : (
-        <div class="text-center text-gray-400">
-          <p>
-            No alumni profiles available at this time. Please check back later!
-          </p>
-        </div>
-      )
-    }
+    {Array.isArray(featuredProfiles) && featuredProfiles.length > 0 ? (
```

Comment:
@SatyaRajAwasth1 instead of this type check and length check

```
Array.isArray(featuredProfiles) && featuredProfiles.length >
```

this single line will do the job